resources/views/hardware/requested.blade.php

```diff
@@ -24,7 +24,7 @@
                        'id' => 'bulkForm']) }}
                     <div class="row">
                         <div class="col-md-12">
-
+                    {{ Form::close() }}
```

Comment:
Was this an accident?

Reply:
No the absence  of a  `Form:close()` was causing a bug with the`Form` facades that I add. I think that probably that is also causing the search functionality you mentioned before.

Reply:
Gotcha. Let me know if you want extra eyes on it 😄